sedbackend/apps/cvs/vcs/storage.py

```diff
@@ -572,6 +571,30 @@ def create_subprocess(db_connection: PooledMySQLConnection, project_id: int, vcs
     return get_subprocess(db_connection, project_id, subprocess_id)
 
 
+def create_multiple_subprocesses(db_connection: PooledMySQLConnection, vcs_id: int,
+                                 subprocesses: List[Tuple[int, models.VCSSubprocessPost]]) -> List[Tuple[int, int]]:
+    logger.debug(f'Creating {len(subprocesses)} subprocesses.')
+
+    if len(subprocesses) == 0:
+        return []
+
+    values = ",".join([f'({vcs_id}, "{subprocess[1].name}", {subprocess[1].parent_process_id})'
```

Comment:
This statement needs to be prepared (https://dev.mysql.com/doc/refman/8.0/en/sql-prepared-statements.html) to be safe. E.g: 

`INSERT INTO {CVS_VCS_SUBPROCESS_TABLE} (vcs, name, iso_process) VALUES (%s, %s, %s)`

You then insert the real values like this:

`cursor.execute(select_stmnt, [vcs_id, subprocess[1].name, subprocess[1].parent_process_id])`

This is critical, as not preparing the statement properly can be a major security risk. Ensure that any manual cursors used (e.g. any place where you have not used the MySQLStatementBuilder package) are properly prepared. Setting cursor(prepared=True) is not enough, you need to change all values to %s.